### DISCUSSION TITLE
What degree of development is planned?

### DISCUSSION THREAD

**Discussion by christophertubbs** [idiap/coqui-ai-TTS#23] [Q&A] [answered]
There are features I can think of that will improve this (such as SSML support). Are there any plans by idiap to continue to add features or is this fork just for minor modifications and keeping things somewhat up to date?

**Comment by eginhard** [comment, marked as answer]
Yes, we do plan to add some new features based on what we develop internally, but it's hard to give an exact roadmap for now. We are actually interested in SSML as well long-term. If you have any ideas for this or other points, feel free to open a separate issue to discuss.

We'll definitely do our best to fix bugs and keep up to date with new package versions. PRs are also very welcome, including for new features and model implementations provided they come with tests and don't add significant maintenance costs, but I'd suggest to first open an issue to discuss any plans.

**Reply by christophertubbs** [reply]
Thank you very much. I really appreciate the work you all are doing.
